data/json/npcs/TALK_ALLY_TUTORIAL.json

```diff
@@ -0,0 +1,198 @@
+      { "text": "Skip it, let's get going.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_ALLY_TUTORIAL_MISC",
+    "type": "talk_topic",
+    "dynamic_line": "You can give me a bunch of instructions.  Talk to me about the miscellaneous rules, and I'll tell you what I'm doing and you can give a new instruction.  Sometimes you'll have shouted a command that overrides your other instructions, and I'll tell you which instructions are overridden. \nI'm not a chatterbox, but I'll tell you when I see or hear danger or if I'm hurt or hungry or something.  You don't want me doing that, you just tell me. \nAlso, I'll pulp zombies if you'd like - I would prefer that. \nIf I hear something suspicious and can't see if, I'll go take a look, but you can tell me not to do that. \n I'll open and close doors, or open them and leave them ajar, or not going through closed doors.  You just tell me what you want. \nI can pick stuff up if want me to, and you can tell me what to pick up.  If I've got a bow or crossbow, please tell me to pick up the ammo - it's less fuss for both of us. \nIf you've got a vehicle, you can e'x'amine it and use the cre'w' command to assign me a seat.  That saves arguments about who needs to sit where.",
+    "responses": [
+      { "text": "How do shouted commands work?", "topic": "TALK_ALLY_TUTORIAL_SHOUTS" },
+      {
+        "text": "Thanks for the explanation.  I wanted to know more about something else.",
+        "topic": "TALK_ALLY_TUTORIAL"
+      },
+      { "text": "Thanks.  I have some things for you to do.", "topic": "TALK_FRIEND" },
+      { "text": "Skip it, let's get going.", "topic": "TALK_DONE" }
+    ]
+  },
+  {
+    "id": "TALK_ALLY_TUTORIAL_COMMANDS",
+    "type": "talk_topic",
+    "dynamic_line": "Here's the list of shouted commands I'll respond to: \nTell me to guard, and I'll stay where I am and watch out for danger.  I'll investigate suspicious noises unless you told me not to. \nTell me to stay away, and I'll stay awake until I'm too tired to keep my eyes open. \nTell me to sleep when tired, and I'll nap when I get tired.  I'll try to tell you when I'm off to bed unless you don't want to hear it. \nTell me to flee, and I'll stick close to you when you run from danger. \nTell me to stop running, and I'll evaluate the danger myself and might go off and fight something while you're doing other things. \nTelling me to prepare for danger is special: it's an override command.  That means I'm going to stuff, even if you've given me other instructions earlier.  Specifically, I'm going to flee with you, I'm not going to open or close doors, I'm not going to sleep until I'm exhausted, and I'm going to try to hold any chokepoints that you're fighting at. \nTelling me to relax from danger is another special command, and it will clear any overrides I currently have and I'll go back to whatever instructions I had before you told me to prepare for danger.",
```

Comment:
> Tell me to stay away, and I'll stay awake until I'm too tired to keep my eyes open.

Should be "Tell me to stay awake"?

> Telling me to prepare for danger is special: it's an override command.  That means I'm going to stuff, 

Missing verb before 'stuff'?

Reply:
Thanks for the suggestions.